### PR TITLE
fix: a11y changes to tabs

### PR DESCRIPTION
**Pull request by stefanoScalzo**
BREAKING CHANGES: change anchor element to button element on tab

## Description
Change the anchor element to a button element
Adds border bottom to remove the white overlapping the container border
fixes the offset on the navigation line to compensate
adds role=link for a11y

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

**Comment by JKMarkowski** [comment]
image
Counters are centered here, could you check it ?

**Comment by JKMarkowski** [comment]
LEt's wait for @jbadan approval

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
The role should not be `link`. Please check again the link provided in the issue description

**Comment by stefanoScalzo** [comment]
> The role should not be `link`. Please check again the link provided in the issue description

fixed all a11y

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
I believe the changes to border bottom are causing this problem with the high contrast version

Screen Shot 2021-01-08 at 9 18 54 AM

**Comment by mikerodonnell89** [comment]
Also I'm still seeing `role="link"` everywhere

**Comment by InnaAtanasova** [comment]
Closing the PR, needs more work

**Pull request closed without merging**
